### PR TITLE
Added getters for name and decimal ERC20

### PR DESCRIPTION
**Pull request by BenKaufman10**
PR for issue #708 
Added name and getters for ERC20 name and decimals.
Made ERC20 decimals public

### - Cute Animal Picture
image

### CODE REVIEW

File: vyper/premade_contracts.py [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 erc20 = """
 class ERC20():
+    def name() -> bytes32: pass
     def symbol() -> bytes32: pass
+    def decimals() -> uint256: pass
```
Comment (jacqueswww) [inline vyper/premade_contracts.py]: @BenKaufman10 please add tests fore the extra two functions.

Comment (jacqueswww) [reply vyper/premade_contracts.py]: I think you can just add some asserts in test_initiat in test_on_chain_market_maker.py

**Comment by fubuloubu** [comment]
I think it's import to note that several of these methods are optional per the spec, although I don't think that matters currently because there should be no harm in adding additional functions to a contract interface as nothing is enforced until the call is made. If the call fails, it's because the contract does not implement the full ERC20 interface.

**Comment by jacqueswww** [comment]
@fubuloubu yes exactly, IMHO the ERC20 shortcut interface should have all the optional functions exposed - for those that want to use it.

File: vyper/premade_contracts.py (hunk 1 above)
Comment (BenKaufman10) [reply vyper/premade_contracts.py]: Alright, I added it there.

**Comment by BenKaufman10** [comment]
@jacqueswww I tried to fix it, you can see in the last commit. For some reason it looks like it didn't check the lates commit (6e17e598906c1f5195c46b753dffe25d75d3035f) which was supposed to fix it.
Can you please check it out and let me know if I missed something?

**Comment by jacqueswww** [comment]
Try reverting your last commit and committing over it, this will trigger the CI again. I don't have permission to retrigger this.

**Comment by BenKaufman10** [comment]
Turns out it was a bug with Travis-ci, just closed the PR and re-opened it and everything worked well.

**Pull request merged**